examples/custom_contexts/commands.py

```diff
@@ -0,0 +1,62 @@
+import typing
+
+import hikari
+
+import crescent
+
+bot = crescent.Bot("...")
+
+# All custom contexts must inherit from `BaseContext``
+# `BaseContext` does not define any methods. It only stores data from the interaction.
+class CustomContext(crescent.BaseContext):
+    def a_custom_method(self) -> str:
+        return "Hello there"
```

Comment:
Is it possible to subclass hikari.Context, to avoid having to call .into()?

If not, I understand this could be because it wouldn't be compatible with autocomplete. But I think it should still be allowed, and just not allowed for autocomplete functions. I think it would be annoying to need to call .into() in order to use other methods.

Reply:
Yes, you can subclass `Context`.

Reply:
I changed the example to show this better.